browser/src/control/jsdialog/Widget.Combobox.js

```diff
@@ -227,10 +227,18 @@ JSDialog.combobox = function (parentContainer, data, builder) {
 	// notebookbar a11y requires main element to have click handler for shortcuts to work
 	container.addEventListener('click', function () { content.focus(); });
 
-	content.addEventListener('keyup', function () {
-		builder.callback('combobox', 'change', data, this.value, builder);
+	content.addEventListener('keyup', function (event) {
+		// special case for address input field combobox in calc.
```

Comment:
Please make this generic. We don't want to hardcode special cases in the standard widget.
Like: in the JSON pass some property, like: `changeOnEnterOnly: true`
Here detect this property and do the job. Avoid hardcoded ids which can be changed later...